lib/kitchen/patches/integer.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# Monkey patches for +Integer+
+#
+class Integer
+  ROMAN_NUMERALS = %w[0 i ii iii iv v vi vii viii ix x].freeze
```

Comment:
How likely will we go beyond 10?

Reply:
extended to 20 just in case, but I think this is unlikely